Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,58 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (ArrayIsNullOrEmpty(array))
```

Comment:
I love the separation of validation and actual function. 2 high level calls- well done.